downstream/assemblies/platform/assembly-example-topologies.adoc

```diff
@@ -1,10 +1,10 @@
 
 ifdef::context[:parent-context: {context}]
 
-[id="assembly-standalone-controller-non-inst-database"]
+[id="assembly-example-topologies"]
 = {AutomationMeshStart} design patterns
 
-:context: standalone-controller-non-inst-database
+:context: assembly-example-topologies
```

Comment:
I'm not a fan of putting `assembly-`, `con-`, `proc-`, or `ref-` in the ID or context because it's just "filler" text and isn't informative to search engines or humans.